Assets/Shopify/UIToolkit/Theme Controllers/ThemeControllerBase.cs

```diff
@@ -65,13 +65,36 @@ public abstract class ThemeControllerBase : MonoBehaviour, IShopCredentials {
         public ShopifyClient Client {
             get {
                 if (_cachedClient == null) {
-                    _cachedClient = new ShopifyClient(LoaderProvider.GetLoader(AccessToken, ShopDomain));
+                    SetupClientAndCart();
                 }
                 return _cachedClient;
             }
         }
         private ShopifyClient _cachedClient;
 
+        public CartController Cart {
+            get {
+                if (_cachedClient == null) {
+                    SetupClientAndCart();
+                }
+
+                return _cachedCart;
+            }
+        }
+
+        private void SetupClientAndCart() {
```

Comment:
Im weary about methods that have a "SomethingAndSomething", maybe its better to call this something along the lines of "InitializeDependencies" or something more generic?

Reply:
I worry that `InitializeDependencies` is too generic